user-guide/Cloud_Platform/CloudAdminApp/Auditing.md

```diff
@@ -9,3 +9,30 @@ If you have the Owner or Admin role for an organization on dataminer.services, y
 This page shows an overview of all the security-related events on dataminer.services that have happened for all systems connected to dataminer.services within your organization.
 
 To quickly find something in the potentially vast number of logs, use the filters at the top of the page.
+
+## Exporting the audit records
+
+<!-- RN 37164 -->
+
+To export the audit records to CSV:
```

Comment:
Maybe mention somewhere the filters that where active for the table, will be applied to the export also (it's told in the popup i think but maybe worth noting here too?)

Reply:
Done!